_includes/sidebar-page.html

```diff
@@ -1,5 +1,5 @@
 <a class="d-block text-decoration-none" href="{{ site.repo }}/edit/{% if site.github_branch %}{{ site.github_branch }}{% else %}master{% endif %}/{{ page.path }}" target="_blank"><i class="bi bi-pencil"></i> Edit this page</a>
 <a class="d-block text-decoration-none" href="{{ site.repo }}/issues/new?labels={% if page.editable %}{{ page.editable }}{% else %}question{% endif %}&title=Question:&body=Question on: {{ page.path }}" target="_blank"><i class="bi bi-github"></i> Create Wiki issue/bug report</a>
-<a class="d-block text-decoration-none" href="{{ page.repo }}/issues/new" target="_blank"><i class="bi bi-list-check"></i> Create Geyser issue/bug report</a>
+<a class="d-block text-decoration-none" href="https://github.com/GeyserMC/Geyser/issues/new/choose" target="_blank"><i class="bi bi-list-check"></i> Create Geyser issue/bug report</a>
```

Comment:
Please revert this, there's a reason we set these links using a variable (so we don't have to update them across all usages)

Reply:
ok